client/state/selectors/has-automated-transfer-eligibility-messages.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import {
+	getAutomatedTransferEligibilityHolds as holds,
+	getAutomatedTransferEligibilityWarnings as warnings,
+} from 'state/selectors';
+
+/**
+ * Determines if there are any eligibility messages to be displayed
+ * for the automated transfer of a site.
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for transfer info
+ * @returns {boolean} True if any warnings or holds exist
+ */
+export default function hasAutomatedTransferEligiblityMessages( state, siteId ) {
+	return !! (
+		holds( state, siteId ).length +
+		warnings( state, siteId ).length
+	);
```

Comment:
just a simple little thought here that we could make this a bit faster (probably doesn't/won't ever matter) by short-circuiting the check. it may actually make this a bit clearer since it looks here like the length of the arrays are important (and they really aren't - only the existence of elements vs empty arrays)

```js
return holds( state, siteId ).length || warnings( state, siteId ).length;
```

in that vein of thought too we might consider using a function call which actually indicates our intents…

```js
return ! ( isEmpty( holds( state, siteId ) ) && isEmpty( warnings( state, siteId ) );
```